lib/metadata_json_deps/runner.rb

```diff
@@ -17,15 +17,25 @@ def initialize(filename, updated_module, updated_module_version, verbose, use_sl
     end
 
     def run
-
       @updated_module = @updated_module.sub('-', '/')
       message = "Comparing modules against *#{@updated_module}* version *#{@updated_module_version}*\n\n"
+      if check_deprecated(@forge.get_current_version(@updated_module)) || @forge.check_deprecated_at(@updated_module)
+        message += "The module you are comparing against #{@updated_module.upcase} is DEPRECATED.\n" #if check_deprecated(@forge.get_current_version(@updated_module)) || @forge.check_deprecated_at(@updated_module)
+        puts message
+        post_to_slack(message) if @use_slack
+        post_to_logs(message) if @logs_file
+        exit
+      end
       @module_names.each do |module_name|
         message += "Checking *#{module_name}*\n"
         metadata = @forge.get_metadata_json(module_name)
+        post_to_logs("\n\n") if @logs_file
```

Comment:
These to `post_to_log` calls are made redundant as they overwrite the file each time they are called. What this means is that the call on line 63 is the only one that actually works. Do we need to have the metadata in the logfile?